### PR TITLE
Contributing the OpenWhisk interactive tutorial to the openwhisk project 

### PR DESCRIPTION
**Pull request by ZmG**

### CODE REVIEW

File: whisk_tutorial/static/js/terminal.coffee [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+  wsk_create_action_hello_v = \
+    """
+    {'apihost': 'openwhisk.ng.bluemix.net', 'namespace': 'jstart', 'clibuild': '2016-03-03T09:55:47-06:00', 'apiversion': 'v1'}
+    ========
+    REQUEST:
+    PUT https://openwhisk.ng.bluemix.net/api/v1/namespaces/jstart/actions/hello
+    Headers sent:
+    {
+        "Authorization": "Basic 
+         UyLWJJkYu65JKhu7YjM0ZDcwODhlNzBiOmlFS3RWMHl0UWdIT1SxUGNrMUFJRHUzSF2VlFSV53hDUnZlVXhyMGJpbTBGeH827=",
```
Comment (rabbah) [inline whisk_tutorial/static/js/terminal.coffee]: you will want to remove this/squash history (and get a new key).

**Comment by mrutkows** [comment]
I do not want to see the history if possible from the precursor project of possible going back to 2013

**Comment by rabbah** [comment]
@mrutkows you could have squashed these commits. Also as a tip, mergers should avoid clicking the merge button on github. It does not preserve a linear history for commits which is a worthwhile feature to have.

@markusthoemmes @csantanapr did we publish our working with git doc to openwhisk yet?

**Comment by mrutkows** [comment]
@rabbah would appreciate pointer on both these instructions.

**Pull request merged**